src/main/webapp/app/entities/iris/iris-session.model.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseEntity } from 'app/shared/model/base-entity';
+import { ProgrammingExercise } from 'app/entities/programming-exercise.model';
+import { User } from 'app/core/user/user.model';
+import { IrisMessage } from 'app/entities/iris/iris-message.model';
+
+export class IrisSession implements BaseEntity {
```

Comment:
Heads up: The server code has changed to support session sub-types now. You don't necessarily have to implement this in this PR yet, but you should handle this in the future.